rpcserverhandlers_test.go

```diff
@@ -3137,6 +3137,124 @@ func TestHandleGetPeerInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleGetStakeVersionInfo(t *testing.T) {
+	t.Parallel()
+
+	blk := dcrutil.NewBlock(&block432100)
+	blkHashString := blk.Hash().String()
+	blkHeight := blk.Height()
+	blk2000Hash := mustParseHash("0000000000000c8a886e3f7c32b1bb08422066dcfd008de596471f11a5aff475")
+	mockVersionCount := []types.VersionCount{{
+		Version: 7,
+		Count:   1,
+	}}
+	defaultChain := defaultMockRPCChain()
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleGetStakeVersionInfo: ok without specifying count",
+		handler: handleGetStakeVersionInfo,
+		cmd:     &types.GetStakeVersionInfoCmd{},
+		result: types.GetStakeVersionInfoResult{
+			CurrentHeight: blkHeight,
+			Hash:          blkHashString,
+			Intervals: []types.VersionInterval{{
+				StartHeight:  defaultChain.calcWantHeight + 1,
+				EndHeight:    blkHeight,
+				PoSVersions:  mockVersionCount,
+				VoteVersions: mockVersionCount,
+			}},
+		},
+	}, {
+		name:    "handleGetStakeVersionInfo: ok with count 2",
+		handler: handleGetStakeVersionInfo,
+		cmd: &types.GetStakeVersionInfoCmd{
+			Count: dcrjson.Int32(2),
+		},
+		result: types.GetStakeVersionInfoResult{
+			CurrentHeight: blkHeight,
+			Hash:          blkHashString,
+			Intervals: []types.VersionInterval{{
+				StartHeight:  defaultChain.calcWantHeight + 1,
+				EndHeight:    blkHeight,
+				PoSVersions:  mockVersionCount,
+				VoteVersions: mockVersionCount,
+			}, {
+				StartHeight: defaultChain.calcWantHeight + 1 -
+					defaultChainParams.StakeVersionInterval,
+				EndHeight:    defaultChain.calcWantHeight + 1,
+				PoSVersions:  mockVersionCount,
+				VoteVersions: mockVersionCount,
+			}},
+		},
+	}, {
+		name:    "handleGetStakeVersionInfo: ok with invalid count",
```

Comment:
Looks like copypasta error here.  It's a failure, not ok.

Reply:
Good catch, updated